orchagent/orchdaemon.cpp

```diff
@@ -84,9 +86,19 @@ bool OrchDaemon::init()
     };
     AclOrch *acl_orch = new AclOrch(m_applDb, acl_tables, gPortsOrch, mirror_orch, neigh_orch, route_orch);
 
-    m_orchList = { gPortsOrch, intfs_orch, neigh_orch, route_orch, copp_orch, tunnel_decap_orch, qos_orch, buffer_orch, mirror_orch, acl_orch, gFdbOrch};
+
+    m_orchList = { gPortsOrch, intfs_orch, neigh_orch, route_orch, copp_orch, tunnel_decap_orch, qos_orch, buffer_orch, mirror_orch, acl_orch, gFdbOrch };
     m_select = new Select();
 
+    vector<string> pfc_wd_tables = {
+        APP_PFC_WD_TABLE_NAME
+    };
+
+    if (platform  == MLNX_PLATFORM_SUBSTRING)
```

Comment:
Remove extra space between `platform` and `==`.

Reply:
Done